src/app/workflow/view/view.component.html

```diff
@@ -17,6 +17,16 @@
 <button type="button" mat-button color="accent" class="btn-block" (click)="showVersionModal()">
   {{ isPublic || !canWrite || (entryType$ | async) === EntryType.Service ? 'View' : 'Edit' }}
 </button>
+<button
+  type="button"
+  mat-button
+  color="accent"
+  class="btn-block dockstore-snapshot"
```

Comment:
If the class is purely used to interact with Cypress, use data-cy (https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements)